controls/radrichtextbox/features/commands.md

```diff
@@ -148,3 +190,170 @@ If you want to handle application commands on RadRichTextBox owner's level, you
 {{endregion}}
 
 {% endif %}
+
+## Command-related Events
+
+**RadRichTextBox** exposes two events that allow you interfere the process of executing a command.
+
+#### [C#] Example 7: Subscribing to the events
+
+{{region radrichtextbox-features-commands_8}}
+	
+	this.radRichTextBox.CommandExecuting += RadRichTextBox_CommandExecuting;
+	this.radRichTextBox.CommandExecuted += RadRichTextBox_CommandExecuted;
+	this.radRichTextBox.CommandError += RadRichTextBox_CommandError;
+{{endregion}}
+
+#### [VB.NET] Example 7: Subscribing to the events
+
+{{region radrichtextbox-features-commands_9}}
+	
+	AddHandler Me.radRichTextBox.CommandExecuting, AddressOf radRichTextBox_CommandExecuting
+	AddHandler Me.radRichTextBox.CommandExecuted, AddressOf RadRichTextBox_CommandExecuted
+	AddHandler Me.radRichTextBox.CommandError, AddressOf RadRichTextBox_CommandError
+{{endregion}}
+
+### CommandExecuting
+
+The **CommandExecuting** event fires just before the execution of a command. You can use it to stop the command or modify its behavior/parameter. The parameters the event enables you to use are of type **CommandExecutingEventArgs** and can give you information about the command the event is fired for and its parameter. 
+
+An example usage of this event is when you need to modify the content that is pasted into the document. In this case, you can easily obtain the content in the clipboard and change it according to your needs just before executing the PasteCommand.  
+
+#### [C#] Example 8: Handling CommandExecuting
+
+{{region radrichtextbox-features-commands_10}}
+
+	void radRichTextBox_CommandExecuting(object sender, Telerik.Windows.Documents.RichTextBoxCommands.CommandExecutingEventArgs e)
+	{
+	    if (e.Command is PasteCommand)
+	    {
+	        // Process the text
+	    }
+	}
+{{endregion}}
+
+#### [VB.NET] Example 8: Handling CommandExecuting
+
+{{region radrichtextbox-features-commands_11}}
+
+	Private Sub radRichTextBox_CommandExecuting(sender As Object, e As Telerik.Windows.Documents.RichTextBoxCommands.CommandExecutingEventArgs)
+	    If (TypeOf (e.Command) Is PasteCommand) Then
+	        'Process the text
+	    End If
+	End Sub
+{{endregion}}
+
+**Example 9** shows how you can stop the execution of a command.

```

Comment:
You don't need this sentence. Generally, I don't like this concept:

*The Example N shows how to do this action*

*#### [C#] Example N: Doing this action.*

You can leave only the caption of the code snippet. If it doesn't look descriptive enough you can change it a bit.